pages/design/tools/index.mdx

```diff
@@ -1,3 +1,8 @@
 # Design tools
 
+Figma is a vital part of Primer Design's workflow. Figma's collaborative environment allows designers to work together across distance and give feedback asynchronously. Learn more about [GitHub's remote design culture here](https://www.figma.com/blog/github-case-study/).
```

Comment:
If we're going to talk about design tools we use I think adding Figma is good but if we are going to talk about design tools we provide we should include prototyping too.

Reply:
Added a sentence about prototyping in the design tools intro text.